documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc

```diff
@@ -0,0 +1,32 @@
+[id='displaying-metrics-state-transfer_{context}']
+= Exporting metrics during a state transfer operation
+
+You can export time metrics for clustered caches that {brandname} redistributes across nodes. 
+
+A state transfer operation occurs when a clustered cache topology changes, such as a node joining or leaving a cluster.
+During a state transfer operation, {brandname} exports metrics from each cache, so that you can determine a cache's status.
+A state transfer exposes attributes as properties, so that {brandname} can export metrics from each cache.
+
+[NOTE]
+====
+You cannot perform a state transfer operation in invalidation mode. 
+====
+
+{brandname} generates time metrics that are compatible with the REST API and the JMX API. 
+
+.Prerequisites
+* Configure {brandname} metrics.
+* Enable metrics for your cache type, such as embedded cache or remote cache.
+* Initiate a state transfer operation by changing your clustered cache topology. 
+
+
+.Procedure
+* Choose one of the following methods:
+** Configure {brandname} to use the REST API to collect metrics.
+** Configure {brandname} to use the JMX API to collect metrics.
+
+[role="_additional-resources"]
+.Additional resources
+* To learn more about enabling statistics in embedded or remote caches, configuring {brandname} metrics, and registering JMX MBeans, see link:{config_docs}#statistics-jmx[Enabling and configuring {brandname} statistics and JMX monitoring ({brandname} caches)].
+
+* To learn more about the state transfer attributes,see link:{javadocroot}/jmxComponents.html#StateTransferManager[`StateTransferManager` ({brandname} {schemaversion} API)].
```

Comment:
Consistency & Jupiter requirements 
No plain paragraphs in the additional resources section. We no longer use this format in our docs.

```suggestion
[role="_additional-resources"]
.Additional resources
* link:{config_docs}#statistics-jmx[Enabling and configuring {brandname} statistics and JMX monitoring]
* link:{javadocroot}/jmxComponents.html#StateTransferManager[`StateTransferManager` ({brandname} {schemaversion} API)]
```

Reply:
Changed link format to match current Jupiter styling.

Reply:
Thanks. Can you also remove the "." in the end?

Reply:
Removed.